File: 프로그래머스/unrated/120887. k의 개수/k의 개수.java
class Solution {
    public int solution(int i, int j, int k) {
        int answer = 0;
        for (int n = i; n <= j; n++) {
            int num = n;
            while (num > 0) {
                int digit = num % 10; 
                if (digit == k) {
                    answer++;
                }
                num /= 10;  
            }
        }
        return answer;
    }
}

File: 프로그래머스/0/120887. k의 개수/k의 개수.js
function solution(i, j, k) {
    var answer = 0;
    for(let n = i; n <=j; n++){
        let temp = n.toString().split('');
        for(let t of temp){
            if(t === k.toString()){
                answer++;
            }
        }
    }
    return answer;
}